Tools/TestWebKitAPI/Tests/WebKitGLib/TestBackForwardList.cpp

```diff
@@ -69,11 +69,10 @@ class BackForwardListTest: public WebViewTest {
         RemovedItems = 1 << 2
     };
 
-    static void checkItem(WebKitBackForwardListItem* item, const char* title, const char* uri, const char* originalURI)
+    static void checkItem(WebKitBackForwardListItem* item, const char* uri, const char* originalURI)
     {
         g_assert_nonnull(item);
         g_assert_cmpstr(webkit_back_forward_list_item_get_uri(item), ==, uri);
-        g_assert_cmpstr(webkit_back_forward_list_item_get_title(item), == , title);
```

Comment:
Instead of removing this line, let's verify that it returns... whatever it returns, presumably nullptr?

Reply:
```
        g_assert_cmpstr(webkit_back_forward_list_item_get_title(item), == , title);
```
> Instead of removing this line, let's verify that it returns... whatever it returns, presumably nullptr?

Restored it and changed it to expect the empty string.